tests/Transports.AspNetCore.Tests/WebSockets/AsyncMessagePumpTests.cs

```diff
@@ -14,7 +16,7 @@ public void CallsSynchronously()
         var sb = new StringBuilder();
         var pump = new AsyncMessagePump<string>(t =>
         {
-            Thread.Sleep(int.Parse(t));
+            Thread.Sleep(int.Parse(t, CultureInfo.InvariantCulture));
```

Comment:
Even these culture additions have extremely limited value. They will not affect production code. They will not affect the tests in the cloud. They can only possibly affect a developer that runs the tests locally with an incompatible culture config. (And presumably you have not had an issue running the tests.)

Reply:
Little benefit is better than nothing. I remember the problems with the culture with the dash sign.

Reply:
I don’t have a problem with this change or this particular analyzer rule, even if it is pointless. I do have a problem with adding all the rest of the analyzer rules. We shouldn’t add the analyzer just to disable 99% of the rules. The analyzers are unnecessary and should not be added.

Reply:
> We shouldn’t add the analyzer just to disable 99% of the rules

6 rules are disabled for tests and 4 for samples. It is far from 99%.